bar-trender-front-end/src/components/ApiFormSubmit.js

```diff
@@ -1,58 +1,77 @@
 import React, { Component } from "react";
-import {  Redirect } from "react-router-dom";
+
 import { Col, Button, Form, FormGroup, Label, Input } from "reactstrap";
 
-    
+
 
 class POSTForm extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            nombre: "",
 
-            filters: {
-
-                billar: "off",
-                paulanner: "off",
-                reina_mercedes: "off",
-                pueblos: "off",
-                ochentero: "off",
-                arabe: "off",
-            },
-        };
+            
+            Paulaner: "off",
+            Triana: "off",
+            Alameda: "off",
+            Cruzcampo: "off",
+            data: null,
         
+        };
 
         this.handleTermChange = this.handleTermChange.bind(this);
         this.handleSearch = this.handleSearch.bind(this);
         this.handleEnter = this.handleEnter.bind(this);
         this.handleSubmit = this.handleSubmit.bind(this);
     }
-
+    
     handleTermChange(e) {
         this.setState({ [e.target.name]: e.target.value });
     }
 
-    async handleSearch() {
+    async handleSearch(e) {
         // this.props.searchEngine(this.state.term);
+
         
-        //alert("A form was submitted: " + JSON.stringify(this.state));
-        this.state = {
-            "filters":{
+        // console.log(this.state.filters);
+        // let temp_state = JSON.stringify(this.state.filters)
+
+        console.log(this.state)
+
+        let beers = [];
+        let zones = [];
+
+        if (this.state['Paulaner'] == "on") {
+            beers.push("Paulaner");
+        }
+        if (this.state['Cruzcampo'] == "on") {
+            beers.push("Cruzcampo");
+        }
+        if (this.state['Alameda'] == "on") {
+            zones.push("Alameda");
+        }
+        if (this.state['Triana'] == "on") {
+            zones.push("Triana");
+        }
+
+         let params = "";
+         beers.map((e,i)=>{
+             if(i===0){
+                 params += "beers="+e;
+             } else{
+                 params += ","+e;
+             }
+         })
+         zones.map((e,i)=>{
+            if(i===0){
+                params += "&zones="+e;
+            } else{
+                params += ","+e;
             }
-        };
-        await fetch('http://localhost:8000/v1/search/establishments', {
-            method: 'POST',
-            // We convert the React state to JSON and send it as the POST body
-            body: JSON.stringify(this.state),
-            headers: {
-                'Content-Type': 'application/json'
-                // Other possible headers
-              }
-          }).then(function(response) {
-            console.log(response)
-            return response.json();
-          });
+        })
+       
+        window.location.href='list?'+params;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7143775)
```suggestion
        window.location.href="list?"+params;
```